bundles/org.eclipse.jface/src/org/eclipse/jface/resource/FileImageDescriptor.java

```diff
@@ -304,12 +303,24 @@ public String getImagePath(int zoom) {
 		return null;
 	}
 
+	private URL toURL() {
+		if (location == null) {
```

Comment:
shou should join both like if (location == null || name == null)  ...

Reply:
Sure, will do.

Reply:
Done.